mapflow/dialogs/main_dialog.py

```diff
@@ -457,6 +466,55 @@ def setup_project_combo(self, projects: dict[str, MapflowProject], current_proje
         if current_project_id:
             self.select_project(current_project_id)
 
+    def setup_projects_table(self, projects: dict[str, MapflowProject]):
+        if not projects:
+            return
+        # First column is ID, hidden; second is name
+        self.projectsTable.setColumnCount(7)
```

Comment:
It's better to have a single source for the numbers. Where does this 7 come from? We must have something in config with the column names